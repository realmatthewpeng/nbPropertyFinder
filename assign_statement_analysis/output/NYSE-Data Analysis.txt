Total number of lines found: 6

data = pd.read_csv("../data/prices.csv")
aapl = data[data.symbol == "AAPL"]
aapl_close = aapl[["close"]]
dates = aapl[["date"]]
aapl_change = aapl_close.apply(lambda x: np.log(x) - np.log(x.shift(1))) # shift moves dates back by 1.
googl = data[data.symbol == "GOOGL"]

Incorrect Lines:


Precision = 1.0
Recall = 1.0

