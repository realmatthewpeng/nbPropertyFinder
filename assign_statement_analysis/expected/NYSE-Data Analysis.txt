data = pd.read_csv("../data/prices.csv")
aapl = data[data.symbol == "AAPL"]
aapl_close = aapl[["close"]]
aapl_change = aapl_close.apply(lambda x: np.log(x) - np.log(x.shift(1))) # shift moves dates back by 1.
dates = aapl[["date"]]
googl = data[data.symbol == "GOOGL"]